### PR TITLE
feat: add new social handles

### PR DESCRIPTION
**Pull request by capJavert**
## Changes

- new social handles
- new validation and parsing
  - validation is on API side now

API PR https://github.com/dailydotdev/daily-api/pull/2076 (also moves validation to API to protect from bad inputs)

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

AS-444 #done

### Preview domain
https://as-444-social-links.preview.app.daily.dev

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Looks good all validation now on api? 👍

**Comment by capJavert** [comment]
> Looks good all validation now on api? 👍

Yes

File: packages/webapp/pages/account/profile.tsx [hunk 1]
```diff
@@ -255,6 +262,17 @@ const AccountProfilePage = (): ReactElement => {
...
+        />
+        <AccountTextField
+          leftIcon={<LinkedInIcon />}
+          label="LinkedIn"
+          inputId="linkedin"
+          hint={hint.linkedin}
+          valid={!hint.linkedin}
+          name="linkedin"
+          value={user?.linkedin}
+          placeholder="username or url"
```
Comment (omBratteng) [inline packages/webapp/pages/account/profile.tsx:275]:
```suggestion
          placeholder="Username or URL"
```
😬

Comment (omBratteng) [reply packages/webapp/pages/account/profile.tsx:275]: On the others too, that are not a url

**Comment by capJavert** [comment]
@nensidosari its not an experiment [see](https://dailydotdev.slack.com/archives/C07BBBA2A3V/p1720519304029029?thread_ts=1720519193.366449&cid=C07BBBA2A3V)

**Comment by nensidosari** [comment]
@capJavert cool, it did cross my mind it could be something like this 😅

**Review by nensidosari — APPROVED** [review]

File: packages/webapp/pages/account/profile.tsx (hunk 1 above)
Comment (capJavert) [reply packages/webapp/pages/account/profile.tsx]: Applied in 9edb90c5222e766f0b0d0642e62a413f98770e7b

**Pull request merged**